source/samples/OpenGLCore/OpenGLCoreSample.cpp

```diff
@@ -63,14 +62,6 @@ int main()
     }
 
     glfwMakeContextCurrent(window);
-    glewExperimental = GL_TRUE;
-    GLuint error;
-    if ((error = glewInit()) != GLEW_OK)
```

Comment:
Hi, I'm not sure I want to move this. The sample are intended as demos for how you would embed Gwork. The initialisation is expected to be done outside of Gwork, and the renderer implementations themselves, again, are only examples.

Reply:
Although it almost never happens, the function pointers can be different across the dll boundary. There is now way to guarantee that the pointers found in the exe are the same as in the dll (in the same way that different opengl context are not guaranteed the same pointer addresses). It is generally better practice to have the dll load its own function pointers rather than.

As long as the opengl context is set to current before calling init this will always work.

Reply:
Just realized it is a static lib (thought it had and option for shared) so the dll stuff might not matter that much. However I  doubt most users are going to write their own renderers until in dire need so I imaging most will go with one of the existing  renderers and it would be better if that renderer did not inject glew into their program.

Reply:
Let me just give you my use case and see if you have another thought. I use glbinding as my extension wrangler. It puts everything into the `namespace gl`. So in order to use it with OpenGLCore I would need to add `using namspace  gl;` to OpenGLCore.cpp which doesn't make sense in the case of a library as not everyone will be using glbinding. So it felt better that the OpenGLCore handle the extensions itself and the user would not need to bother.